hw_1/task_3.py

```diff
@@ -0,0 +1,33 @@
+# Напишите код, который запрашивает число и сообщает является ли оно простым или составным.
+# Используйте правило для проверки: «Число является простым, если делится нацело только на единицу
+# и на себя». Сделайте ограничение на ввод отрицательных чисел и чисел больше 100 тысяч.
+MIN_LIMIT = 0
+MAX_LIMIT = 100_000
+
+
+def digit_check():
+    while True:
+        try:
+            num_int = int(input('Введите число от 1 до 100 000: \n')) 
+            if  num_int > MIN_LIMIT and num_int < MAX_LIMIT:
+                return num_int
+        except ValueError:
+            print('Введите число от 1 до 100_000:\n')
+            continue
+
+def simple_num(num):
+    count = 0
+    for i in range(2, num ):
+        if num % i == 0:
+            count +=1
+    if count > 2:
+        return False
+    else:
+        return True
```

Comment:
В переменной `count ` нет никакой необходимости, ведь как только интерпретатор зайдет в 22 строку, это будет означать, что число составное и в дальнейших итерациях нет никакого смысла. Используйте связку `break` и `else` от цикла с выводом соответствующего сообщения и все. Но так как у вас функция , то можно еще проще
```suggestion
            return False

    return True
```